Kentor.AuthServices.Owin/KentorAuthServicesAuthenticationOptions.cs

```diff
@@ -75,5 +81,23 @@ public string Caption
                 Description.Caption = value;
             }
         }
+
+        private void FederationIdentityConfiguration(FederationIdentityConfiguration identity)
+        {
+            federationIdentityConfiguration = identity;
+        }
+
+        /// <summary>
+        /// The federation identity configuration. 
+        /// </summary>
+        public IdentityConfiguration IdentityConfiguration
```

Comment:
I want a better name, to make it clear that this relates to System.IdentityModel and not AuthServices' own federation element.